cypress/support/pages/home.pageObject.js

```diff
@@ -4,7 +4,47 @@ class HomePageObject extends PageObject {
   url = '/#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return cy.getByDataQa('username-link');
+  }
+
+  assertUsername(username) {
+    this.usernameLink.should('contain', username);
+  }
+
+  get globalFeed() {
+    return cy.getByDataQa('Global-Feed');
+  }
+
+  get NoArticleMessage() {
+    return cy.getByDataQa('no-article');
+  }
+
+  assertAfterDeleteArticle() {
+    this.globalFeed.should('contain', 'Global Feed');
+    this.NoArticleMessage.should('contain', 'No articles are here... yet.');
+  }
+
+  assertLogOUtHomePage() {
```

Comment:
The method name 'assertLogOUtHomePage' should follow camel case naming convention. Consider renaming it to 'assertLogOutHomePage'.